ntoskrnl/kd64/kdapi.c

```diff
@@ -2199,15 +2199,15 @@ KdSystemDebugControl(
                     break;
 
             }
-            break;
+            return STATUS_SUCCESS;
```

Comment:
Is this something the sanitizer triggered ?

Reply:
No, I just use `KdSystemDebugControl` in ubsan.c for printing a stack trace, and it always printed "KdSystemDebugControl is unimplemented!" after the stack, which does not make sense